exercises/anagram/canonical-data.json

```diff
@@ -132,6 +132,17 @@
       },
       "expected": []
     },
+    {
+      "uuid": "630abb71-a94e-4715-8395-179ec1df9f91",
+      "reimplements": "7cc195ad-e3c7-44ee-9fd2-d3c344806a2c",
+      "description": "does not detect an anagram if the original word is repeated",
+      "property": "findAnagrams",
+      "input": {
+        "subject": "go",
+        "candidates": ["goGoGO"]
+      },
+      "expected": []
+    },
     {
       "uuid": "9878a1c9-d6ea-4235-ae51-3ea2befd6842",
       "description": "anagrams must use all letters exactly once",
```

Comment:
Should we state "all letters of the source word" or something similar?

Reply:
Isn't that a different test case? I mean, I don't disagree, but it looks like a different test case?

Reply:
Does that mean that the description, then, is not right in general?

Reply:
I think so

Reply:
@kotp Did you mean change "anagrams must use all letters exactly once" to "anagrams must use all letters of the source word exactly once"? As @ErikSchierboom says this is the testcase below. Or did you mean change "does not detect an anagram if the original word is repeated" to "anagrams must use all letters of the source word exactly once"?

If you meant the former, is the description ok as it is now? It currently is "does not detect an anagram if the original word is repeated", which seems to fit the test case even after removing the spaces, since it is the original word being repeated, just without spaces in-between.

Reply:
I think your change is fine on lines 135-145, so we are really talking about what you have not changed, and may change, as suggested.

Additionally, if you choose not to fix it while we are here, that is OK too.  It may be changed at another time, in another pull request.

Reply:
Well, I must say that your change may be redundant if the following test case is described better.  It is not described well at the moment.  It may be enforcing the same thing.

Reply:
It depends on how one looks at it I guess. The original reason for this change was that the existing test is problematic due to the whitespace. I can change the next test's description as suggested if you prefer, or close the pull request and leave the goGoGO test as  it is until it can be deprecated/removed completely.

Reply:
Let's merge this as is and do the update of the other test case in a separate PR.